mitmproxy/web/app.py

```diff
@@ -315,10 +318,19 @@ def put(self):
                 self.master.server.config.http2 = v
                 update[k] = v
             elif k == "anticache":
-                self.master.anticache = v
+                self.master.options.anticache = v
                 update[k] = v
             elif k == "anticomp":
-                self.master.anticomp = v
+                self.master.options.anticomp = v
+                update[k] = v
+            elif k == "stickycookie":
+                self.master.set_stickycookie(v)
+                update[k] = v
+            elif k == "stickyauth":
+                self.master.set_stickyauth(v)
+                update[k] = v
+            elif k == "stream":
+                #self.master.set_stream_large_bodies(v) todo: ask
```

Comment:
can anybody tell me how to set stream, should i set 
-- self.master.stream 
-- self.master.stream_large_bodies
-- ...

thanks